lib/node_modules/@stdlib/stats/base/dists/studentized-range/quantile/benchmark/benchmark.js

```diff
@@ -21,27 +21,36 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pkg = require( './../package.json' ).name;
 var quantile = require( './../lib' );
+var uniform = require('@stdlib/random/base/uniform');
+var Float64Array = require( '@stdlib/array/float64' );
```

Comment:
```suggestion
var isnan = require( '@stdlib/math/base/assert/is-nan' );
var uniform = require( '@stdlib/random/base/uniform' );
var Float64Array = require( '@stdlib/array/float64' );
var pkg = require( './../package.json' ).name;
var quantile = require( './../lib' );
```

Just some rearrangement for consistency.